nexus-constructor.py

```diff
@@ -22,6 +23,12 @@
     root_dir = os.path.dirname(os.path.realpath(__file__))
 
 if __name__ == "__main__":
+    if locale.getlocale()[0] is None:
+        used_locale = "en_GB.UTF-8"
```

Comment:
setting the locale here is a good idea. Just for curiosity, did you face an issue with opening file or something?

Reply:
This actually came from another fix that did not make it into this PR but I thought that I might as well put the set-locale-code here anyway.